lib/features/thread/presentation/widgets/email_tile_builder.dart

```diff
@@ -143,6 +146,17 @@ class EmailTileBuilder with BaseEmailItemTile {
     );
   }
 
+  Widget _buildCalenderIcon(PresentationEmail email) {
+    String mainString = email.emailHeader.toString();
+    String substring = "null";
+
+    if (mainString.contains(substring)) {
+      return buildIconCalendar( email);
```

Comment:
What is the different between `buildIconCalendar` and `_buildCalenderIcon`? How about more meaningful method name?